src/_blog/rust-libp2p-browser-to-server.md

```diff
@@ -0,0 +1,61 @@
+---
+tags:
+  - rust-libp2p
+  - rust
+  - libp2p
+  - browser
+  - webrtc
+  - webassembly
+  - wasm
+title: Rust-libp2p in the Browser with WebRTC!
+description: News about how rust-libp2p now connects with the network via WebRTC
```

Comment:
```suggestion
description: News about how rust-libp2p in the browser connects to nodes in a network using WebRTC
```

I suggest:

- Not talking about _the_ network. I.e. there are many.
- Emphasizing that rust-libp2p **in the browser** is the special part here.

Don't feel strongly about the exact wording.